PSPrintSDK/OLKiteViewController.m

```diff
@@ -97,13 +97,13 @@ - (void)transitionToNextScreen:(BOOL)animated{
         }
         else{
             for (OLProduct *productIter in [OLProduct products]){
-                if (productIter.templateType == self.templateType){
+                if ([productIter.productTemplate.identifier isEqualToString:product.productTemplate.identifier]){
```

Comment:
Is product not guaranteed to always be nil here? Or is it just too late on a Friday and I'm not able to reason logically anymore :)